### PR TITLE
Remove remote icon

### LINKED ISSUE #5: Icon is broken on flathub website
This should either not use the `remote` icon or install the icon in the flatpak itself. 

That way it should show up fine then.

### PR DESCRIPTION
**Pull request by bbhtt**
Closes https://github.com/flathub/com.airtame.Client/issues/5

### CODE REVIEW

**Comment by klausenbusk** [comment]
@bbhtt do you want to become a maintainer of this flatpak? :)

**Comment by bbhtt** [comment]
> @bbhtt do you want to become a maintainer of this flatpak? :)

Thank you for the offer, but I'd have to decline. Since I don't use this regularly I might not be able to respond to issues or  make fixes all the time. I'll still send occasional PRs if needed/if I can. Thanks again :)

File: com.airtame.Client.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```
Comment (klausenbusk) [inline com.airtame.Client.desktop:1]: What is the rationale for creating our own desktop file?

**Comment by klausenbusk** [comment]
> Thank you for the offer, but I'd have to decline. Since I don't use this regularly I might not be able to respond to issues or make fixes all the time. I'll still send occasional PRs if needed/if I can. Thanks again :)

I can relate ;) I haven't used Airtame for years and is very too slow to respond to issues/PRs.

File: com.airtame.Client.desktop (hunk 1 above)
Comment (bbhtt) [reply com.airtame.Client.desktop:1]: I think the appstream check reported some issues a while back when I did several of these. I'll check again.

Comment (bbhtt) [reply com.airtame.Client.desktop:1]: Yea, if the desktop is placed in export/ in the extra-data step, looks like, it looks for the icons there but can't find any.

**Pull request merged**